elastic/kibana
# [Response Ops] API to retrieve execution log entries from event log.

Resolves https://github.com/elastic/kibana/issues/126837

## Summary

Adds new HTTP API to retrieve execution log information.

## To Verify
* Create a rule and let it run.
* Navigate to `https://localhost:5601/internal/alerting/rule/<rule_id>/_execution_log?date_start=<ISO date string>` and see entries for the execution log. Only `date_start` query parameter is required, all others have default values. Defaults to sorting by descending timestamp order with 10 results per page.
* Try passing in other query params like sort, page, and per page.
* Response should look like this:

```
{
      total: 374,
      data: [
        {
          id: '6705da7d-2635-499d-a6a8-1aee1ae1eac9',
          timestamp: '2022-03-07T15:38:32.617Z',
          duration_ms: 1056,
          status: 'success',
          message:
            "rule executed: example.always-firing:a348a740-9e2c-11ec-bd64-774ed95c43ef: 'test rule'",
          num_active_alerts: 5,
          num_new_alerts: 5,
          num_recovered_alerts: 0,
          num_triggered_actions: 5,
          num_succeeded_actions: 5,
          num_errored_actions: 0,
          total_search_duration_ms: 0,
          es_search_duration_ms: 0,
          timed_out: false,
          schedule_delay_ms: 3393,
        },
        {
          id: '41b2755e-765a-4044-9745-b03875d5e79a',
          timestamp: '2022-03-07T15:39:05.604Z',
          duration_ms: 1165,
          status: 'success',
          message:
            "rule executed: example.always-firing:a348a740-9e2c-11ec-bd64-774ed95c43ef: 'test rule'",
          num_active_alerts: 5,
          num_new_alerts: 5,
          num_recovered_alerts: 5,
          num_triggered_actions: 5,
          num_succeeded_actions: 5,
          num_errored_actions: 0,
          total_search_duration_ms: 0,
          es_search_duration_ms: 0,
          timed_out: false,
          schedule_delay_ms: 3456,
        },
      ],
    }
```

### Checklist
